modules/openid_auth/app/controllers/openid_auth/v0/mvi_users_controller.rb

```diff
@@ -6,22 +6,32 @@ module OpenidAuth
   module V0
     class MviUsersController < ApplicationController
       skip_before_action :authenticate
-      before_action :check_required_headers
+      before_action :check_required_headers, only: :show
 
-      # Allows MVI lookups based on user identity traits (presumably obtained from the upstream
-      # identity provider). Returns a 200 response when the MVI lookup succeeds. Raises an error
-      # otherwise.
+      def search
+        user_attributes = JSON.parse(request.body.read, symbolize_names: true)
```

Comment:
Rails should place the body in the params object. Since the body has no root key rails should place the body in the params object with a key based on the controller name, 'mvi_users'. So you can remove JSON.prase, replacing it with `user_attributes = params[:mvi_users]`. See https://edgeapi.rubyonrails.org/classes/ActionController/ParamsWrapper.html